docs/source/en/model_doc/conditional_detr.md

```diff
@@ -43,7 +43,6 @@ This model was contributed by [DepuMeng](https://huggingface.co/DepuMeng). The o
 
 [[autodoc]] ConditionalDetrImageProcessor
     - preprocess
-    - pad_and_create_pixel_mask
```

Comment:
I think we don't need it, but still wondering why we don't put `pad` here.

Reply:
It's just for consistency with other image processors, where we only list the `preprocess` method in the docs